components/staticFeeds/sample-messages.json

```diff
@@ -39,7 +39,7 @@
       "goLiveDate": "2017-08-03",
       "expireDate": null,
       "featureImageUrl": null,
-      "priority": "high",
+      "priority": null,
```

Comment:
Turning off high priority here inconsistent with `title` of this sample notification.